index.js

```diff
@@ -99,4 +99,17 @@ module.exports = class BemEntityName {
     isEqual(entity) {
         return entity && (this.id === entity.id);
     }
+
+    /**
+     * Determines whether specified entity belongs to this.
+     * @param {object} entity - the entity to compare.
+     * @returns {boolean}
+     */
+    belongsTo(entity) {
```

Comment:
I think `contains` or `includes` not suitable for `BemEntityName`.

Such methods can be in `BemEntity`.